### DISCUSSION TITLE
Welcome to archlinux-installer Discussions!

### DISCUSSION THREAD

**Discussion by kwankiu** [kwankiu/archlinux-installer-rock5#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by endeavouros-arm** [comment]
I have a couple of suggestions for your install script.  

1 before starting any package downloading, in /etc/pacman.d/mirrorlist, allow the user to comment out the 
Geo-IP based mirror selection and load balancing
entry and un-comment mirrors closest to them.

2 before starting any package downloading, in /etc/pacman.config, allow the user to uncomment 
 ParallelDownloads 
and set the number of desired Parallel Downloads they desire.

This will increase the speed and reliability of package downloads during the install.

https://github.com/endeavouros-arm/image-build-script/blob/main/config-eos.sh

Lines 20 to 130 in the above script preforms these two functions.  This is using whiptail instead of ncurses? 
Feel free to use any of the above code if you decide to incorporate the above enhancements.

**Reply by kwankiu** [reply]
> I have a couple of suggestions for your install script.

Thanks for the suggestions.

> 1 before starting any package downloading, in /etc/pacman.d/mirrorlist, allow the user to comment out the Geo-IP based mirror selection and load balancing entry and un-comment mirrors closest to them.

I am working on this, on the dev branch I added an option to edit mirrorlist, I am also planning on ranking the mirrors and automatically picking the most suitable mirror.

> 2 before starting any package downloading, in /etc/pacman.config, allow the user to uncomment ParallelDownloads and set the number of desired Parallel Downloads they desire.
> 
> This will increase the speed and reliability of package downloads during the install.
> 
It is currently setting to 50 by default on b07 (was 30 on b05/b06), on the dev branch, there's now an option to edit pacman.conf

**Comment by endeavouros-arm** [comment]
Sounds like you have it covered.

Pudge

**Comment by nrpetonr** [comment]
great job @kwankiu. testing w/ SDcard on orangepi 5plus w/ kde desktop, so far so goood. i even consider to make it my main build, therefore i am wondering whether it can be copied to nvme, e.g. with acu ,to speed it up further.

**Reply by nrpetonr** [reply]
meanwhile did a clean install to nvme. as expected, it is flying now :-)

**Comment by DarkevilPT** [comment]
Hey mi Kumpa I have found this youtubee script for installing arch linux..
And maybe u can get some ideas that u could adapt/adopt to improve your next versions of Arch Linux Installer.

So here https://youtu.be/PqGnlEmfYjM?si=5CFRzWQHLz7mPEUf
